addons/website/views/website_visitor_views.xml

```diff
@@ -111,7 +112,7 @@
                                         </div>
                                         <div>
                                             <img t-if="record.country_id.raw_value"
-                                             t-att-src="kanban_image('res.country', 'image', record.country_id.raw_value)"
+                                             t-att-src="record.country_flag.raw_value"
```

Comment:
Couldn't we use the kanban widget ?

Reply:
Nope, `image_url` (and `image`) widgets are totally broken when rendered raw in a kanban card...
This will need further investigation with the Js team and left like that currently...